### DISCUSSION TITLE
Embed an iFrame in UI

### DISCUSSION THREAD

**Discussion by cryptosystems6300** [windmill-labs/windmill#3834] [Questions]
When I include an iFrame in the HTML element, I get the error: _This content is blocked. Contact the site owner to fix the issue._

How can I embed iFrames into my UI?

![image](https://github.com/windmill-labs/windmill/assets/110590814/82f17d01-fa21-48e6-899f-5582e8b74bb3)

**Comment by anurag472** [comment]
you have to make changes in x-frame-option on the load balancer for the other site
